plugins/ssl/letsencrypt_weekly

```diff
@@ -41,6 +90,15 @@ warning=${warning:-:40}
 critical=${critical:-:50} #letsencrypt doesn't allow more than 50 certificates per week
 # see https://letsencrypt.org/docs/rate-limits/
 
+
+get_files_and_domains() {
+	find /etc/letsencrypt/csr/ -mtime -7 -type f -print0 2>/dev/null | xargs -0 -I pem bash -c 'echo -n "pem "; openssl req -in pem -text -noout | grep DNS: | sed "s/.*DNS://g"'
```

Comment:
`--no-run-if-empty` would be suitable here (for `xargs`), or?

Reply:
when using `-I ...` you do not need `--no-run-if-empty` because `-I` can only convert the variable to a value if a value is found, if no value is found it does not invoke the command with empty argument:
```bash
set -x
find /tmp -maxdepth 1 -name propably_not_existing -print0 | xargs -0 -I element echo element
```
shows no invocation of echo:
```
+ xargs -0 -I element echo element
+ find /tmp -maxdepth 1 -name propably_not_existing -print0
```